### DISCUSSION TITLE
how to fix exit code 255

### DISCUSSION THREAD

**Discussion by dwnsideup** [FabricMC/fabric-api#1597] [Player support]
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'imm_ptl_core'!
Exit Code: 255

**Comment by warjort** [comment]
imm_ptl_core is immersive portals
https://github.com/qouteall/ImmersivePortalsMod/issues

**Reply by dwnsideup** [reply]
Thanks, I found out that removing sodium fixed it, so I'm guessing sodium and immersive portals clash.
